package.json

```diff
@@ -42,53 +42,108 @@
         "configuration": {
             "title": "Competitive Programming Helper",
```

Comment:
Changes : cph.setting -> cph.general/language.setting, removed language ids, added compiler drop downs and added menuChoices setting.

Reply:
That's a great idea! :D